codice/ddf
# DDF-4606 Clean up blueprint definitions for Catalog UI Search

#### What does this PR do?
Arranges the Catalog UI Search wiring to be organized by (1) coupling levels to DDF mapped to files, and (2) files arranged by app or feature. 

Future changes will now clearly reveal if the UI's coupling to DDF is increasing or decreasing. 

#### Who is reviewing it? 
@rymach 
@gjvera 
@mojogitoverhere 

#### Select relevant component teams: 
@codice/ui 

#### Ask 2 committers to review/merge the PR and tag them here.
@djblue 
@rzwiefel 

#### How should this be tested?
Verify the system starts up, there are no errors in the logs, there are no errors on bundle refresh, and Intrigue is working as expected:
1. Verify the ability to create, edit, and delete workspaces. 
1. Verify the ability to query the system (if one query succeeds, the app is wired together correctly).
1. Verify the ability to create, edit, and delete search forms.
1. Verify the ability to share a workspace or search form. 
1. Verify the ability to use Intrigue's uploader: upload any XML or TXT file and ensure a wildcard search yields a result for each uploaded file. 
1. Verify the historian can be toggled on and off: run `log:set TRACE org.codice.ddf.catalog.ui.config` and refresh the catalog UI search bundle, repeating the refresh as necessary with the **Catalog > Configuration > Historian** checkbox toggled off / on (default for DDF should be on) to verify the log changes. 

#### Any background context you want to provide?
This change is a simple, relatively low-risk effort to help make maintenance of UI features more manageable. 

#### What are the relevant tickets?
Fixes: #4606 

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
